pgpool/deployment/overlays/production/pgpool_configmap.yaml

```diff
@@ -34,10 +34,10 @@ data:
     memqcache_method = 'shmem'
     memqcache_total_size = 512MB
     memqcache_max_num_cache = 10000
-    memqcache_expire = 14400
+    memqcache_expire = 86400
     memqcache_auto_cache_invalidation = off
     memqcache_cache_block_size = 1MB
-    log_min_messages = 'DEBUG5'
+    log_min_messages = 'WARNING'
```

Comment:
DEBUG5 emits more detailed debug messages to the log compared to WARNING. Will changing this not affect the debugging process?

Reply:
Because the pgpool has been working for a long time without any problem, so I consider we can elevate the log to save some resources.